### DISCUSSION TITLE
Automatically Fetch Tables

### DISCUSSION THREAD

**Discussion by sonny-mei** [Budibase/budibase#14682] [Help] [answered]
Hello,

My Budibase instance is connected to an MS SQL table, and I want to keep the tables up-to-date in Budibase. Currently, I have to manually re-fetch the tables to ensure any deleted columns are removed. Is there a way to automatically refresh the tables or an API that can handle the re-fetching process?

![image](https://github.com/user-attachments/assets/c046dddc-1f2d-4cca-b2d1-d95904850a02)

Thanks!

**Comment by mike12345567** [comment, marked as answer]
Hi @sonny-mei, this is not something that is currently supported, once a Budibase app is published we assume that the schema of the data will not change. If the structure of your data changes rapidly you may need to use queries rather than the table integrations.
